flaky/flaky_decorator.py

```diff
@@ -39,7 +54,7 @@ def flaky(max_runs=None, min_passes=None):
     if max_runs < min_passes:
```

Comment:
Unrelated to this change. I'd move L51-53 to the top of this method

   if hasattr(__call__):
      wrapper = max_runs
      max_runs = None

Then the comment explaining that code is simpler. And there's no duplication of applying a special default value of 2.